File: add/codesnippet/CPP/m-system.net.webclient.d_2_1.cpp
      Console::Write( "\nPlease enter a URI (e.g. http://www.contoso.com): " );
      String^ remoteUri = Console::ReadLine();
      
      // Create a new WebClient instance.
      WebClient^ myWebClient = gcnew WebClient;
      // Download home page data.
      Console::WriteLine( "Downloading {0}", remoteUri );
      // Download the Web resource and save it into a data buffer.
      array<Byte>^ myDataBuffer = myWebClient->DownloadData( remoteUri );
      
      // Display the downloaded data.
      String^ download = Encoding::ASCII->GetString( myDataBuffer );
      Console::WriteLine( download );

      Console::WriteLine( "Download successful." );

File: add/codesnippet/CSharp/m-system.net.webclient.d_2_1.cs
            Console.Write("\nPlease enter a URI (for example, http://www.contoso.com): ");
            string remoteUri = Console.ReadLine();

            // Create a new WebClient instance.
            WebClient myWebClient = new WebClient();
            // Download home page data.
            Console.WriteLine("Downloading " + remoteUri);                        
            // Download the Web resource and save it into a data buffer.
            byte[] myDataBuffer = myWebClient.DownloadData (remoteUri);

            // Display the downloaded data.
            string download = Encoding.ASCII.GetString(myDataBuffer);
            Console.WriteLine(download);
                                
            Console.WriteLine("Download successful.");